### DISCUSSION TITLE
能将一些经常遇到的问题或者注意点在这里归档吗？方便后来学习者

### DISCUSSION THREAD

**Discussion by jiaoqingbo** [apache/celeborn#1273] [Q&A]
能将一些经常遇到的问题或者注意点在这里归档吗？方便后来学习者

**Comment by FMX** [comment]
这里可以加入钉钉群（41594456）我们或者 [Slack 频道](https://join.slack.com/t/apachecelebor-kw08030/shared_invite/zt-1ju3hd5j8-4Z5keMdzpcVMspe4UJzF4Q)

**Comment by yanfangwang** [comment]
你好，我加入这个群发现失效，有更新吗

**Reply by FMX** [reply]
https://qr.dingtalk.com/action/joingroup?code=v1,k1,/mQqdQFqW8AeP8CbLqiZUlbmMdg2smv0HN8gf8nMf6M=&_dt_no_comment=1&origin=11?  明济邀请你加入钉钉群聊Celeborn用户群，点击进入查看详情
